patient-search/ui/src/pages/patientProfile/style.scss

```diff
@@ -1,9 +1,33 @@
 .delete-btn {
```

Comment:
I think important! will become difficult to handle later down the line. You might be able to provide a stricter selector other than just .delete-btn as a work around. Fine for this PR though.